plugin/src/main/java/at/helpch/chatchat/command/IgnoreListCommand.java

```diff
@@ -0,0 +1,40 @@
+package at.helpch.chatchat.command;
+
+import at.helpch.chatchat.ChatChatPlugin;
+import at.helpch.chatchat.api.user.ChatUser;
+import dev.triumphteam.cmd.bukkit.annotation.Permission;
+import dev.triumphteam.cmd.core.BaseCommand;
+import dev.triumphteam.cmd.core.annotation.Command;
+import dev.triumphteam.cmd.core.annotation.Default;
+import org.bukkit.Bukkit;
+import org.bukkit.OfflinePlayer;
+
+import java.util.stream.Collectors;
+
+@Command("ignorelist")
+public class IgnoreListCommand extends BaseCommand {
+    private final ChatChatPlugin plugin;
+    private final static String IGNORELIST_PERMISSION = "chatchat.ignoreList";
+
+    public IgnoreListCommand(final ChatChatPlugin plugin) {
+        this.plugin = plugin;
+    }
+
+    @Permission(IGNORELIST_PERMISSION)
+    @Default
```

Comment:
Annotations should probably be ordered alphabetically.

Reply:
idk, I just copied ignore command and changed it to be a different command